src/pocketmine/level/generator/normal/biome/SwampBiome.php

```diff
@@ -20,23 +20,26 @@
 */
 
 namespace pocketmine\level\generator\normal\biome;
-
+use pocketmine\level\generator\populator\Flower;
+use pocketmine\block\Block;
+use pocketmine\block\Flower as FlowerBlock;
 class SwampBiome extends GrassyBiome{
-
 	public function __construct(){
 		parent::__construct();
-
+		
+		$flower = new Flower();
+		$flower->setBaseAmount(8);
+		$flower->addType([Block::RED_FLOWER, FlowerBlock::TYPE_BLUE_ORCHID]);
+		
+		$this->addPopulator($flower);
 		$this->setElevation(62, 63);
-
 		$this->temperature = 0.8;
 		$this->rainfall = 0.9;
 	}
-
-	public function getName() : string{
+	public function getName(){
```

Comment:
I modified it to my server's needs. One of the plugins was spamming the console with an error associated with it. :/